molo/core/models.py

```diff
@@ -279,6 +317,11 @@ class SiteSettings(BaseSetting):
         null=True,
         blank=True,
     )
+    section_ordering = enum.EnumField(
```

Comment:
can you add help text so admins are clear on the difference between the two?

Reply:
This should be one field called 'Article ordering within sections'

Reply:
![ArticleOrderingScreenshot_2019-04-16](https://user-images.githubusercontent.com/5039730/56225633-0fda1700-6072-11e9-8712-5ce2a12de6e5.png)